wagtailio/core/migrations/0061_create_initial_sectors.py

```diff
@@ -0,0 +1,30 @@
+from django.db import migrations
+
+SECTOR_DATA = (
+    "Government agencies",
+    "Corporations",
+    "Education",
+    "Cultural institutions",
+    "Charities and nonprofits",
+)
+
+
+def _populate_sector_data(apps, schema_editor):
+    Sector = apps.get_model("core", "Sector")
+    to_create = []
+    for name in SECTOR_DATA:
+        to_create.append(Sector(name=name))
+    Sector.objects.bulk_create(to_create)
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("core", "0060_sector_showcaseitem_showcasepage"),
+    ]
+
+    operations = [
+        migrations.RunPython(
+            code=_populate_sector_data,
+            reverse_code=migrations.RunPython.noop,
```

Comment:
question: why `noop`?

Reply:
I've added a backwards method. The answer is the previous migration removes this model so I guess I just thought to remove migrate backwards is the same when you include the scope of the PR.